ndcube/ndcube.py

```diff
@@ -341,6 +345,15 @@ def axis_world_coords(self, *axes):
         cube_dimensions = np.array(self.dimensions.value, dtype=int)
         n_dimensions = cube_dimensions.size
         world_axis_types = self.world_axis_physical_types
+        # Extract the kwarg `edges` if present.
+        # True signifies pixel_edges
+        # False signifies pixel_values
+        # Default value is False
+        if "edges" in kwargs:
```

Comment:
A quicker way to do this is `kwargs.get("edges", False)`.

Reply:
Sure, making it right away!